### ISSUE TITLE
Question: Logs

### ISSUE THREAD

**Issue by LaocheXe** [e107inc/e107#3960]
The site I am working on had a strange Parse error displaying, broke the website. 

Parse error: syntax error, unexpected end of file, expecting ')' in /e107/e107_system/xxxxxx/logs/logi_264.2019.php on line 163

easy fix was to delete the php file.

I however, made a copy before deleting it.

Looks like the file wasn't complete as line 163 is     'date' of an array.

`  array (
    'host' => 'static.belong.com.au',
    'date' => 1569129779,
    'os' => 'MacOS X',
    'browser' => 'Chrome v76.0.3809.',
    'screen' => '1920x1080@24',
    'referer' => 'https://www.501st.com/forum/viewtopic.php?f=11&t=104463',
  ),
  'pool-173-75-37-185.pitbpa.fios.verizon.net' => 
  array (
    'host' => 'pool-173-75-37-185.pitbpa.fios.verizon.net',
    'date'`

From line 152 to 163 - first time this happen, don't know if it's something I should worry about happening again or not.

**Comment by Moc** [comment]
Yeah it's a bug that needs fixing. I have had it before but was never able to reproduce it.

**Comment by Moc** [comment]
@LaocheXe Do you still have the entire copy of the file?

**Comment by LaocheXe** [comment]
@Moc Yes I do

https://gist.github.com/LaocheXe/378fb1fcdfcf3915772d2fcd7175ff6f
